projects/README.md

```diff
@@ -2,16 +2,15 @@
 
 |project|license|description|version|repository|patch/diff|
 |-------|-------|-----------|-------|----------|----------|
+|[CPPZMQ](https://github.com/zeromq/cppzmq 'CPPZMQ')|[MIT](https://github.com/zeromq/cppzmq/blob/v4.2.1/LICENSE 'MIT')|CPPZMQ is a minimal c++ binding to the libzmq functions.|4.2.1|none|none|
 |[FFmpeg](https://www.ffmpeg.org/ 'FFmpeg website')|[LGPL](https://www.ffmpeg.org/legal.html 'Lesser GPL v2.1')|complete, cross-platform solution to record, convert and stream audio and video|2.6.2|[repo](https://github.com/distributepro/FFmpeg 'forked FFmpeg repo on github')|[diff](https://github.com/distributepro/FFmpeg/compare/FFmpeg:n2.6.2...xp2.6.2 'patch/diff')|
```

Comment:
I would say this is a good change to move CPPZMQ here to the beginning so it's alphabetical but I think you should probably revert the rest of the changes to this file, mostly it looks to be removing the manually added diff portions.